hedera-node/src/main/java/com/hedera/services/txns/contract/process/EvmTxProcessor.java

```diff
@@ -107,6 +111,16 @@ public EvmTxProcessor(
 		operationRegistry.put(new HederaSelfDestructOperation(gasCalculator));
 		/* Deregister CREATE2 Opcode */
 		operationRegistry.put(new InvalidOperation(0xF5, gasCalculator));
+
+		operationRegistry.put(new HederaBalanceOperation(gasCalculator));
```

Comment:
Nit: I'd move these next to `HederaCreateOperation` and `HederaSelfDestruct` operation, rather than nesting the CREATE2 removal in the middle.